### ISSUE TITLE
Write a small Node app using Webpack & ES6

### ISSUE THREAD

**Issue by stickel** [dotsara/100-projects#7]
Build a basic Node app. Add Webpack to use in the build process and transpile some ES6 code using Babel.

**Comment by dotsara** [comment]
First step… look up all those words. (Except Node, I know that one.) (Okay, and ES6.)

Okay, so first step: look up Webpack…

**Comment by dotsara** [comment]
[what is webpack](https://webpack.github.io/docs/what-is-webpack.html) is surprisingly unhelpful. I'm going to generously posit that's because I haven't really worked with Node, yet. 

Well, except for the [track your damn time](https://github.com/latentflip/track-your-damn-time) Node app I used to use (but which broke when I migrated to a new machine; sadtrombone).

**Comment by stickel** [comment]
Yeah, the docs on webpack are pretty thick. Look up tutorials on what it is outside of the official docs.
